grails-app/controllers/mama/ng/scheduler/UserController.groovy

```diff
@@ -0,0 +1,126 @@
+package mama.ng.scheduler
+
+import grails.converters.JSON
+import org.codehaus.groovy.grails.web.json.JSONObject
+
+class UserController {
+
+    static allowedMethods = [save: "POST", update: "PUT", delete: "DELETE"]
+
+
+    /**
+     * Pagination params:
+     * max = max items to return; default 10; max 100
```

Comment:
The default and maximum are described as `10` and `100` here but the code seems to use `50` and `200`? Same comment applies to the other `max` documentation.